helix-core/src/test/java/org/apache/helix/integration/rebalancer/TestAutoRebalancePartitionLimit.java

```diff
@@ -204,10 +215,14 @@ public boolean verify() {
           cache.getStateModelDef(cache.getIdealState(_resourceName).getStateModelDefRef())
               .getStatesPriorityList().get(0);
       int replicas = Integer.parseInt(cache.getIdealState(_resourceName).getReplicas());
-      return verifyBalanceExternalView(
-          accessor.getProperty(keyBuilder.externalView(_resourceName)).getRecord(),
-          numberOfPartitions, masterValue, replicas, cache.getLiveInstances().size(),
-          cache.getIdealState(_resourceName).getMaxPartitionsPerInstance());
+      try {
+        return verifyBalanceExternalView(
+            accessor.getProperty(keyBuilder.externalView(_resourceName)).getRecord(),
+            numberOfPartitions, masterValue, replicas, cache.getLiveInstances().size(),
+            cache.getIdealState(_resourceName).getMaxPartitionsPerInstance());
+      } catch (Exception e) {
+        return false;
```

Comment:
add a log.error please. It is better with stack trace. This would be tremendous helpful if the test fails.

Reply:
Let me add a debug log here.
1. It is expected. Since the EV might not be created when the verify starts. So if we output an error message, it only pollutes the log with no good reason.
2. It will be retried in the loop. It would be too verbose if we output on every checks.